tests/integration/test.issue7810.js

```diff
@@ -0,0 +1,39 @@
+describe('#7810 query will not find documents when an index is used', function () {
+    beforeEach(function () {
+       return new PouchDB('mydb').destroy().then(function () {
+            return new PouchDB('mydb', { db: require('memdown') }).destroy();
+        });
+    });
+    it('should find the documents even if a index is used', function () {
+        PouchDB.plugin(require('../../packages/node_modules/pouchdb-find'));
+        var CustomPouch = PouchDB.defaults({db: require('memdown')});
+        var db = new CustomPouch('mydb');
+        return db.createIndex({
+            index: {
+                fields: [
+                    'passportId'
+                ]
+            }
+        }).then(function () {
+            return db.put({
+                _id: 'foobar',
+                passportId: 'z3i7q29g4yr1',
+                firstName: 'Edison',
+                lastName: 'Keebler',
+                age: 24
+            });
+        }).then(function () {
+            return db.find({
```

Comment:
I don't quite follow this test. So why do you use `memdown`? 
I can't see how `_id: {}` would cause it to use the above defined index?

Reply:
I use the memory adapter because I copied that from another test in the suit. It does not matter which adapter is used for this bug.
The query does intentional NOT use the index. The bug is that if the index exists but is not used, this has a side-effekt on the query which then returns the wrong result. See #7810

Reply:
Is it for any selector or only for `_id: {}` as the selector?

Reply:
It is for any selector which does not trigger the usage of the index

Reply:
Is there any information missing? @garrensmith